subsys/bluetooth/mesh/net.c

```diff
@@ -1400,6 +1407,19 @@ void bt_mesh_net_start(void)
 
 		bt_mesh_prov_complete(net_idx, addr);
 	}
+
+#if defined(CONFIG_BT_MESH_PROVISIONER)
+	struct bt_mesh_node *node;
+	int i;
+
+	for (i = 0; i < ARRAY_SIZE(bt_mesh.nodes); ++i) {
+		node = &bt_mesh.nodes[i];
+		if (node->net_idx != BT_MESH_KEY_UNUSED) {
+			bt_mesh_prov_node_added(node->net_idx, node->addr,
+						   node->num_elem);
```

Comment:
I'm not sure I agree with this addition. I'd expect the provisioner app to use this callback as a trigger to start configuration of the node, something you'd only really want to do once. Wouldn't it make more sense for the app to do this on its own in init if it needs to? Currently, there's no trivial way to tell whether this callback is coming for the first time, or if it's just a repeat of a previously added node.

Reply:
Uhm yea I think I just added it because that's how the `complete()` callback for the local node is done. I will remove it.

Currently there is no way for an application to actually get a hold of the provisioned nodes either but I was thinking that I would add that in another PR.

Reply:
Fixed